src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/JsonTest/JsonBulkCopyTest.cs

```diff
@@ -23,14 +23,17 @@ public JsonBulkCopyTest(ITestOutputHelper output)
             _output = output;
         }
 
-        private void PopulateData(int noOfRecords)
+        private void PopulateData(int noOfRecords, int rows)
         {
             using (SqlConnection connection = new SqlConnection(DataTestUtility.TCPConnectionString))
             {
                 DataTestUtility.CreateTable(connection, "jsonTab", "(data json)");
                 DataTestUtility.CreateTable(connection, "jsonTabCopy", "(data json)");
```

Comment:
Please don't use any hardcoded table names. Use utility functions to generate object names instead.